dasharo-compatibility/sata-detect.robot

```diff
@@ -0,0 +1,59 @@
+*** Settings ***
+Library             Collections
+Library             OperatingSystem
+Library             Process
+Library             String
+Library             Telnet    timeout=20 seconds    connection_timeout=120 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+Resource            ../sonoff-rest-api/sonoff-api.robot
+Resource            ../rtectrl-rest-api/rtectrl.robot
+Resource            ../variables.robot
+Resource            ../keywords.robot
+Resource            ../keys.robot
+
+Suite Setup         Run Keyword
+...                     Prepare Test Suite
+Suite Teardown      Run Keyword
+...                     Log Out And Close Connection
+
+
+*** Test Cases ***
+SAT001.001 SATA should be visible from OS
+    [Documentation]    This test aims to verify that SATA is detected from OS
+    ...    by using smartctl.
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    SAT001.002 not supported
+    Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    SAT001.002 not supported
+
+    Power On
+    Boot System Or From Connected Disk    ubuntu
+    Login To Linux
+    Switch To Root User
+    Detect Or Install Package    smartmontools
+
+    ${lsblk_out}=    Execute Command In Terminal    lsblk -d -o NAME -n
+    @{disks}=    Split String    ${lsblk_out}    \n
+
+    FOR    ${disk}    IN    @{disks}
+        ${out}=    Execute Command In Terminal    sudo smartctl -i /dev/${disk}
+        Log    ${out}
+        ${sata_present}=    Run Keyword And Return Status    Should Contain    ${out}    SATA Version is:
+        Pass Execution If    '${sata_present}' == 'TRUE'    'SATA disk found, passing test'
+    END
+
+SAT001.002 SATA should be visible from OS
```

Comment:
Do we want to use the same names for different cases?

Reply:
@macpijan fixed it in [this commit](https://github.com/Dasharo/open-source-firmware-validation/pull/290/commits/a9e1586aff88c185035f5690b13aef0df869c9bc)